conda/base/constants.py

```diff
@@ -97,6 +96,8 @@
     "win-arm64",
     "zos-z",
 )
+KNOWN_SUBDIRS: Final = ("noarch",)
+KNOWN_SUBDIRS += PLATFORMS
```

Comment:
```suggestion
KNOWN_SUBDIRS: Final = ("noarch", *PLATFORMS)
```

The type checkers will complain otherwise